lib/features/email/presentation/controller/single_email_controller.dart

```diff
@@ -122,6 +122,7 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
   final eventActions = <EventAction>[].obs;
   final emailLoadedViewState = Rx<Either<Failure, Success>>(Right(UIState.idle));
   final emailUnsubscribe = Rxn<EmailUnsubscribe>();
+  final attachmentsViewState = RxMap<Id, Object>();
```

Comment:
why not `RxMap<Id, Either<Failure, Success>>>();`?

Reply:
Done